src/Tracker/TrackerClient.php

```diff
@@ -259,7 +259,7 @@ private function setDaysSinceLastSession( &$event ) {
         sdk::debug('setting days since last session.', $this->getSetting('debug'));
         $dsps = '';
         if ( $this->getGlobalEventProperty( 'is_new_session' ) ) {
-            sdk::debug( 'timestamp: ' . $event->get( 'timestamp' ) );
+            sdk::debug( 'timestamp: ' . $event->get( 'timestamp' ), $this->getSetting('debug') );
```

Comment:
Not sure this is best way to handle this. How about we move the check for $this->getSetting('debug') into the actual sdk::debug method. That way we can turn it off everywhere.